gito/bootstrap.py

```
@@ -25,23 +27,41 @@ def format(self, record):
-    setup_logging()
-    logging.info("Bootstrapping...")
+    log_levels_by_verbosity = {
+        0: logging.CRITICAL,
+        1: logging.INFO,
+        2: logging.DEBUG,
+        3: logging.DEBUG,
+    }
+    Env.verbosity = verbosity
+    Env.logging_level = log_levels_by_verbosity.get(verbosity, logging.INFO)
+    setup_logging(Env.logging_level)
+    logging.info(f"Bootstrapping... "+mc.ui.gray(f"[verbosity={verbosity}]"))
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)